### ISSUE TITLE
Adding random colors

### ISSUE THREAD

**Issue by mprimeau1** [Montana-Media-Arts/120_CreativeCoding#173]
So I am trying to make it so that my ellipse randomly changes colors between 3 different colors. I can make the ellipses change color if I type in red or blue or whatever or if I input a random color number lets say 75.  But when I use my color picker it won't work. I can use the color picker for background colors and other solid non-random things, but not when I want to randomize colors to change. Any ideas?

**Comment by DVSnell** [comment]
I did this for Week 5 and it worked pretty well.

First, I made the "random" color strings live in an array:
`let colors = ['rgb(255, 0, 0)', 'rgb(0, 255, 0)', 'rgb(0, 0, 255)', 'rgb(255, 255, 255)', 'rgb(0, 0, 0)'];`

It's key to include the apostrophes as that will tell p5 that these are strings and not numerical values.

Then I called it in my draw function with:
`fill(random(colors));`
which pulls a random value (or string in this case) from the array named 'colors' and plugs it into the fill or stroke, apostrophes and all. This is important since the color picker must use strings.
